railties/lib/rails/generators/rails/app/templates/gitignore

```diff
@@ -17,5 +17,6 @@
 /log/*
 <% if keeps? -%>
 !/log/.keep
+!/tmp/.keep
```

Comment:
We need to actually generate this file, grep the codebase for `empty_directory_with_keep_file` (probably need to add a test too)